Docs: Debugging steps for data plane (#8944)
solo-io/gloo: docs/content/operations/debugging_gloo/_index.md

## Debugging the data plane

Gloo Edge is based on Envoy proxies. If requests are handled incorrectly, use the following `glooctl` CLI tools to debug your Envoy configuration.  

### General steps

1. Find the resources that report errors. You can use the [`glooctl check` command](#glooctl-check) to check all of your resources or get the details of the virtual service, gateway, or route table that behave incorrectly. Check if the resource shows any errors in their status. 
   ```sh
   glooctl check
   kubectl get virtualservice <virtualservice-name> -n <namespace> -o yaml
   kubectl get routetable <routetable-name> -n <namespace> -o yaml
   kubectl get gateway <gateway-name> -n <namespace> -o yaml
   ```

> 2. If the resources seem to be ok, you can check any referenced upstreams to verify that they are configured correctly. Things to check include the label selectors to make sure the correct app service is selected, the port that the app serves on, and the IP address of the app service that the upstream points to. 

Reviewer: Maybe turn into bullets so it's more like a checklist?
```suggestion
2. If the resources seem to be ok, you can check any referenced upstreams to verify that they are configured correctly. In particular, check that the following settings are correct:
   * Label selectors for the app service
   * The port that the app serves on
   * The IP address of the app service that the upstream points to
```

glooctl proxy stats -n gloo-system --name internal-proxy | grep -i default-httpbin-8000
```

### View Envoy bootstrap config and access the Admin API

> There may be more limited times where you need direct access to the [Envoy Admin API](https://www.envoyproxy.io/docs/envoy/latest/operations/admin). You can view both the Envoy bootstrap config as well as access the [Admin API](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) with the following commands:

Reviewer: Maybe rephrase since you're already updating the page
```suggestion
In limited cases, you might need direct access to the [Envoy Admin API](https://www.envoyproxy.io/docs/envoy/latest/operations/admin). You can view both the Envoy bootstrap config as well as access the [Admin API](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) with the following commands:
```
